app/assets/javascripts/registerSecurityKey.js

```diff
@@ -0,0 +1,56 @@
+(function(window) {
+  "use strict";
+
+  window.GOVUK.Modules.RegisterSecurityKey = function() {
+    this.start = function(component) {
+      $(component)
```

Comment:
I should have mentioned this before but can we put in a guard at the top to make it return if the needed web APIs aren't available?
```suggestion
      if (!('fetch' in window) || !('credentials' in window.navigator)) { return }
      $(component)
```

Reply:
We did [discuss it before](https://github.com/alphagov/notifications-admin/pull/3878#pullrequestreview-657903585), but not from a JS point of view. I'll leave it as-is for now, and make it part of the follow-up PR so it's all in one place (I've made a not of your suggested change on the card).